torch/nn/quantized/_reference/modules/conv.py

```diff
@@ -0,0 +1,136 @@
+import torch
+import torch.nn.quantized as nnq
+import torch.nn.functional as F
+from typing import Optional
+from torch.nn.common_types import _size_1_t
+from torch.nn.modules.utils import _single
+
+class _ConvNd(nnq._ConvNd):
```

Comment:
I don't feel too strongly one way or the other and haven't thought through it in detail, just wondering, do we want reference modules to inherit from quantized modules, or do we want the reverse?  I would have expected the reverse

Reply:
I see, yeah I think it probably makes more sense to do the reverse

Reply:
let me do it in a different PR, we need to change linear as well

Reply:
sounds good